samples/azure/azure-service-bus-long-running/ASB_7/Processor/Processor.cs

```diff
@@ -0,0 +1,136 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.WindowsAzure.Storage.Table;
+using NServiceBus;
+using NServiceBus.Logging;
+using Shared;
+
+public class Processor
+{
+    static ILog log = LogManager.GetLogger<Processor>();
+    IEndpointInstance endpoint;
+    CloudTable table;
+
+    static readonly ConcurrentQueue<RequestRecord> toProcess = new ConcurrentQueue<RequestRecord>();
+
+    public void Start(IEndpointInstance endpointInstance, CancellationToken token)
+    {
+        endpoint = endpointInstance;
+        var cloudTableClient = StorageHelper.GetTableClient();
+        table = cloudTableClient.GetTableReference(Constants.TableName);
+        table.CreateIfNotExists();
+
+        var task = new Task(() =>
```

Comment:
@danielmarbach what is the best way of doing the below? ie start two tasks and return immediately